src/main/updates/update.tsx

```diff
@@ -0,0 +1,19 @@
+import { updateElectronApp, UpdateSourceType } from 'update-electron-app';
+import electron, { autoUpdater } from 'electron';
+
+export function startAutoUpdate() {
+  // Handle update-downloaded event
+  autoUpdater.on('update-downloaded', (event, releaseNotes, releaseName) => {
+    // Yolo I guess 🤷‍♀️
+    autoUpdater.quitAndInstall();
```

Comment:
This totally works for now, in the future we might want to hold an update till (for example) the current match finishes